src/test/java/io/jenkins/plugins/jfrog/configuration/JFrogPlatformInstanceTest.java

```diff
@@ -0,0 +1,56 @@
+package io.jenkins.plugins.jfrog.configuration;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author yahavi
+ **/
+public class JFrogPlatformInstanceTest {
+
+    private static final String DISTRIBUTION_URL = "https://acme.jfrog.io/distribution";
+    private static final String ARTIFACTORY_URL = "https://acme.jfrog.io/artifactory";
+    private static final String XRAY_URL = "https://acme.jfrog.io/xray";
+    private static final String URL = "https://acme.jfrog.io";
+    private static final String SERVER_ID = "acme";
+
+    @Test
+    public void testInferArtifactoryUrl() {
+        // Check that Artifactory URL inferred from the platform URL
+        JFrogPlatformInstance jfrogPlatformInstance = new JFrogPlatformInstance(SERVER_ID, URL, null, "", "", "");
+        assertEquals("", jfrogPlatformInstance.getArtifactoryUrl());
+        assertEquals(ARTIFACTORY_URL, jfrogPlatformInstance.inferArtifactoryUrl());
+
+        // Check that Artifactory URL does not inferred from the platform URL
```

Comment:
...does not... --> ...is not...
Is this description correct? Are we checking the Artifactory URL is not inferred from the platform URL, or that it is inferred?

The above teo comments also apply to the below two methods.

Reply:
Yes, this description is correct -
The test sets the platform URL to an empty string and Artifactory URL to https://acme.jfrog.io/artifactory.
The test verifies that the empty platform URL does not override the Artifactory URL.